news/2022-08-01-osumonthly-1.md

```diff
@@ -0,0 +1,211 @@
+- [Top Scores of the Month](#top-scores-of-the-month)
+- [New features on osu!lazer (& bonus article!)](#new-features-on-osulazer)
+- [Summary of this article](#summary-of-this-article)
+- [End Word](#end-word)
+
+---
+
+## What's up on osu!? 
+- *Latest updates, events… and even more!*
+
+### ![](/wiki/shared/mode/osu.png) osu! 
+#### If you ask us what is the big event of July, it'd be the **Roundtable**!
+<br>
+
+![](/wiki/shared/news/2022-08-01-osumonthly-1/roundtable/RTable_title.png)
+![](/wiki/shared/news/2022-08-01-osumonthly-1/roundtable/RTable_players.jpg)
+
+With the players showcased above, we have [BTMC](https://osu.ppy.sh/users/3171691) and [Happystick](https://osu.ppy.sh/users/256802) commentating, a genius roster who make sense for this first **IRL osu! invitational**! 
+
+As if it wasn't enough, [mrekk](https://osu.ppy.sh/users/7562902) was there too! If you weren't able to follow the event, no worries! The official Roundtable's YouTube channel has the full VODs [here](https://www.youtube.com/channel/UC6fSQd4oKBPxkBR93GucCUg/featured)!
+
+With that many great players, such as [mrekk](https://osu.ppy.sh/users/7562902), [Utami](https://osu.ppy.sh/users/7512553), [RyuK](https://osu.ppy.sh/users/6304246) and many others, this tournament was incredible! The competition's spirit was mixed with the entertaining cast and gimmicks of [BTMC](https://osu.ppy.sh/users/3171691) and the result was epic!
+
+In terms of epic, we were served with the finals : [mrekk](https://osu.ppy.sh/users/7562902) vs [Utami](https://osu.ppy.sh/users/7512553)! Despite mrekk's defeat against Utami, his performances were impressive, along with Utami's. His first rank on the leaderboard is indeed his throne.
+
+A best-of from this match was made on the BTMC channel, with exclusive interviews of the players, by BTMC (what a surprise, ahah) and Happystick, the casters of this amazing show! You can find it [here](https://www.youtube.com/watch?v=Y7LqBZL)!
+
+Anyway, The Roundtable has now ended, and we wait impatiently the next edition, next year maybe ?
+
+[➥ *Find the forum article here!*](https://osu.ppy.sh/home/news/2022-07-08-the-roundtable) 
+
+Thanks again to [BTMC](https://osu.ppy.sh/users/3171691) and his staff for this insane event! 
+
+### ![](/wiki/shared/mode/mania.png) osu!mania
+#### In August, the osu!mania top players will brawl in the *osu!mania World Cup 2022*!
+
+As it is starting the **6th of August**, we can't actually talk about it more, but we are excited to say that the next osu!monthly is going to have a **big article** about it, and maybe an **interview of the winner**!
+
+Please stay tuned about the next article, and we're moving on the next section!
+
+[➥ *Find the forum article here!*](https://osu.ppy.sh/home/news/2022-07-07-osu-mania-4k-world-cup-2022-registrations-now-open) 
+
+
+### ![](/wiki/shared/mode/catch.png) osu!catch
+#### At the beginning of July, the recent eleventh iteration of osu!catch World Cup has been concluded.
```

Comment:
```suggestion

*At the beginning of July, the recent eleventh iteration of osu!catch World Cup has been concluded.*
```
Seems a bit long for a title to me. And it's the only title for this section anyway.